src/github.com/stellar/federation/record.go

```diff
@@ -1,8 +1,12 @@
 package federation
 
-type Record struct {
-	Username  string `db:"username" json:"stellar_address"`
-	AccountId string `db:"account_id" json:"account_id"`
-	MemoType  string `db:"memo_type" json:"memo_type,omitempty"`
```

Comment:
We can change it to `StellarAddress` if you want. Then for federation result:
* `AccountId` in `Record` => `id` from DB result
* `MemoType` in `Record` => `memo_type` from DB result (or empty)
* `Memo` in `Record` => `memo` from DB result (or empty)

For reverse federation:
* `StellarAddress` in `Record` => `name` from DB result

And `json` tags are the same for both types.